prog/vm/nexus.rb

```diff
@@ -305,15 +306,14 @@ def before_run
 
   label def wait
     when_start_after_host_reboot_set? do
+      register_deadline(:wait, 5 * 60)
```

Comment:
I will test it. I'm uncertain about what will occur when we set a deadline for the current label.

Reply:
I tested it